.circleci/config.yml

```diff
@@ -281,7 +281,7 @@ defaults:
   - base_ems_large: &base_ems_large
       docker:
         - image: << pipeline.parameters.emscripten-docker-image >>
-      resource_class: large
+      resource_class: macos.m1.large.gen1
```

Comment:
This is wrong. Those images are not osx-based. The only image that need to be changed are the `base_osx_large` as far as I know. But we can also change the `base_osx` to `macos.x86.medium.gen2` just to be explicit about the resource, although this will be the default if no resource class is specified.

Reply:
I had my suspicion as well regarding this. Glad that you caught it. Apologies about the mistakes.

Reply:
Do I have to revert all the changes for the new medium tags as well? Because I just did that in my latest push.

Reply:
No, now it is good. Just squash your commits to a single one ;)